gateone/i18n/fr_FR/LC_MESSAGES/gateone_js.po

```diff
@@ -61,37 +61,37 @@ msgstr "Aller"
 
 #: gateone/applications/terminal/plugins/ssh/static/ssh.js:495
 msgid "Are you sure you wish to delete this identity?"
-msgstr "Êtes-vous sûr de vouloir supprimer cette identité?"
+msgstr "Êtes-vous sûr de vouloir supprimer cette identité ?"
 
 #: gateone/applications/terminal/plugins/ssh/static/ssh.js:497
 msgid ""
 "An X.509 certificate may be uploaded to add to this identity.  If one "
 "already exists, the existing certificate will be overwritten."
-msgstr "Un certificat X.509 peut être téléchargé à ajouter à cette identité. S'il en existe déjà, le certificat existant sera remplacé."
+msgstr "Un certificat X.509 peut être transféré pour l'ajouter à cette identité. S'il en existe déjà, le certificat existant sera remplacé."
 
 #: gateone/applications/terminal/plugins/ssh/static/ssh.js:986
 msgid ""
 "The private key for this SSH identity is protected by a passphrase.  "
 "Please enter the passphrase so a public key can be generated."
-msgstr "La clé privée pour cette identité SSH est protégé par un mot de passe. S'il vous plaît entrez le mot de passe si une clé publique peut être généré."
+msgstr "La clé privée pour cette identité SSH est protégé par un mot de passe. Veuillez entrez le mot de passe pour qu'une clé publique puisse être généré."
 
 #: gateone/applications/terminal/plugins/ssh/static/ssh.js:987
 msgid ""
 "<b>NOTE:</b> This passphrase will only be used to extract the public key "
 "and will not be stored."
-msgstr "<b>Remarque:</b> Ce mot de passe ne sera utilisé pour extraire la clé publique et ne sera pas enregistrée."
+msgstr "<b>Remarque:</b> Ce mot de passe ne sera utilisé que pour extraire la clé publique et ne sera pas enregistrée."
 
 #: gateone/applications/terminal/plugins/ssh/static/ssh.js:988
 msgid "Passphrase"
 msgstr "Phrase de passe"
 
 #: gateone/applications/terminal/static/terminal.js:87
 msgid "Terminal output has been suspended (Ctrl-S). Type Ctrl-Q to resume."
-msgstr "La borne de sortie a été suspendu (Ctrl-S). Tapez Ctrl-Q pour le reprendre."
+msgstr "La sortie de la console a été suspendu (Ctrl-S). Tapez Ctrl-Q pour reprendre."
 
 #: gateone/applications/terminal/static/terminal.js:125
 msgid "WARNING: Problematic Browser Extension Detected"
-msgstr "AVERTISSEMENT: Problématique Browser Extension détecté"
+msgstr "AVERTISSEMENT: Problématique avec l'extension du navigateur détecté"
```

Comment:
I would have translated it by:
"AVERTISSEMENT: Extension du navigateur défectueuse détectée !"